make/mt-config/tests/mt-tests-fuzz.mk

```diff
@@ -49,13 +49,15 @@ clang_compiler: add_clang_to_path
 
 #### =========================================================
 
-XI_LIBFUZZER_URL := https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer
+# XI_LIBFUZZER_URL := https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer
```

Comment:
This is total separated issue from the ESP32.
Meanwile our Travis CI is down or the public one fails always for STM32 and CC3200 builds one of our other target started to fail. This wasn't recognised since the build fails anyways, nobody pays attention, understandably. Actually what happened is only a prerequisite repository of a fuzztes target was moved. And git repo wasn't simply moved but also converted to an SVN repo.

Reply:
But the content remained the same.

Reply:
Ahh, gotcha :+1:

Reply:
removed the old one, thx 🔭 well spotted

Reply:
I can still see it there. Did you forget to commit the file or something?

Anyway, this shouldn't hold up the PR. Feel free to ignore these comments and merge; I'm just being nitpicky.